recipes-bsp/u-boot/u-boot-iot2050.inc

```diff
@@ -78,6 +78,10 @@ dpkg_runbuild_prepend() {
     export TEE=/usr/lib/optee-os/iot2050/tee-pager_v2.bin
 }
 
+dpkg_runbuild_prepend_secureboot() {
+    export FIRMWARE_SECURE_VER="${FIRMWARE_SECURE_VER}"
```

Comment:
We will have to change this for sbuild compatibility, but - as I learned recently - issues in meta-coral will not allow us to pick that up very soon anyway. So it's fine for now.